src/components/Avatar/Avatar.scss

```
@@ -77,7 +77,7 @@ $large-size: rem(60px);
   left: 50%;
   width: 100%;
   height: 100%;
-  background-color: var(--p-neutral, color('sky'));
+  background-color: var(--p-secondary, color('sky'));
```

We should use this pull request as an opportunity to get rid of the base (unmodified) variants. They've been more confusing than useful.

Update this to use the disabled secondary action like `Badge` uses.

```suggestion
background-color: var(--p-action-secondary-disabled, color('sky'));
```

---

Okay, double checked this was the only one